Libraries/libutil/Sources/SysUtil.cpp

```diff
@@ -52,12 +52,7 @@ EnvironmentVariables()
 }
 
 #if defined(__linux__)
-static char initialWorkingDirectory[PATH_MAX] = { 0 };
-__attribute__((constructor))
-static void InitializeInitialWorkingDirectory()
-{
-    getcwd(initialWorkingDirectory, sizeof(initialWorkingDirectory));
-}
+static std::string initialWorkingDirectory = FSUtil::GetCurrentDirectory();
```

Comment:
Should I replace it with a GNU constructor? It's basically the same, except more verbose and less portable.